stack.yaml

```diff
@@ -1,7 +1,7 @@
 # Note: for now only GHC API 7.10 is supported, and this is resolver is the
 # latest corresponding one which has an fpco/stack-build docker image. See
 # below why is that useful.
-resolver: lts-6.30
+resolver: lts-8.17
```

Comment:
please fork file to stack-8.17.yaml (and rename other to -6.30.yaml), and either have a local symlink to your desired one, or use some stack env var to select the one.